### DISCUSSION TITLE
local build image for testing source changes

### DISCUSSION THREAD

**Discussion by czaban** [docker-library/postgres#1197] [General]
I've copy the Dockerfile and docker-ensure-initdb.sh and docker-entrypoint.sh from 16/bullseye and build it: 
docker build -t <private registry address>/postgresqlbullseye:latest .
docker push <private registry address>/postgresqlbullseye:latest

After docker run I am getting error: /usr/bin/env: ‘bash\r’: No such file or directory
the same story when I test it with 14/alpine3.19

Where is the root cause, please?

**Comment by tianon** [comment]
That's definitely going to be Windows line endings (the `\r` is the clue -- Windows typically uses `\r\n` and Linux/Unix uses just `\n`).

**Reply by tianon** [reply]
See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings for some decent GitHub documentation about line endings.

**Reply by czaban** [reply]
You're right - that was it, building under Windows and the Git feature. Thank you very much.
